### ISSUE TITLE
me too,have u figured out any solution?i've got an flow graph ,but it failed to run,maybe we can discuss and help each other

### ISSUE THREAD

**Issue by viola-glitch** [sacinold/channel-sounding#1]

**Comment by sacinold** [comment]
I have gotten the flow graph and believe it worked well. The problem I am facing now is the conversion of .dat file (from file sink) at the receiver end of the channel sounding to a readable data using gnu octave.
 
![Uploadin
![sliding correlator send tx grc grc](https://user-images.githubusercontent.com/60781607/78988072-69f3a800-7b1f-11ea-836c-d8da2ad7706a.png)
g sliding correlator send tx.grc.grc.png…]( 

![ sldc with waterfall grc](https://user-images.githubusercontent.com/60781607/78988387-51d05880-7b20-11ea-848c-73c682aabaa8.png)
)
The transmitter up and the receiver down. Can you try it?

**Comment by viola-glitch** [comment]
i've tried the solution in csdn,but it failed.BTW,i am curious about how could u manage a channel sounder without USRP,and my instructor told me to change the title.do u have the same question?

**Comment by sacinold** [comment]
viola-glitch,
Good day. I hope you're enjoying this compulsory holiday. I am using NI
USRP-2920 for both transmitter and receiver.

**Comment by viola-glitch** [comment]
feel happy for u.Good night!

**Comment by sacinold** [comment]
viola-glitch,
Are u saying you do not have access to USRP now?

**Comment by viola-glitch** [comment]
yeah,so i am really confused now.besides,i don't  know how to get the PDP from the CIR either.have u firgure out the "file sink "problem?

**Comment by viola-glitch** [comment]
BTW ,my flow graph is as below:
![2020-04-17 13-52-22 的屏幕截图](https://user-images.githubusercontent.com/59651968/79536462-b0fb1380-80b2-11ea-94bc-e7fc4f82d610.png)
the result is :
![Uploading 2020-04-17 13-53-32 的屏幕截图.png…]()
the CIR,but i don't know how to deal with it

**Comment by viola-glitch** [comment]
i saw some channel sounder has the PLL module,and i am afraid that the PN sequences in the tx and rx are not  all the same?how is everything going in your labratory?are you still in the school now?

**Comment by sacinold** [comment]
I reside on campus so it is easy for me to work in the Laboratory at this
period. Did you see my flow graphs I sent to you sometimes ago? They are
the ones I am using for the project.

**Comment by sacinold** [comment]
I have not yet figured out the "file sink. still working on it.

**Comment by viola-glitch** [comment]
https://github.com/gbaier/gr_channelsounder
Have u tried this project?i have tried this in my labratory 3 months ago,and it worked well ,i can see the CIR ,PDP (time variable)but  be aware that your gnuradio version must be 3.7.2 ,mine is 3.7.11,now it doesn't work,cause the qt binding of my version is pyqt5,while the required version is pyqt4,which is compatible with 3.7.2
